mods/noita-mp/.gitignore

```diff
@@ -0,0 +1,4 @@
+/luarocks.bat
```

Comment:
@ofoxsmith your LuaRocks set up isn't correct. We should fix this. This should be removed. The files as well.

Reply:
I followed the process dictated in the building readme, and all this was generated by it.

Reply:
I think you missed this
```
REM // cd fullPathTo\NoitaMP\mods\noita-mp
set INIT_PATH=%CD%
```
Your LuaRocks stuff is inside root and not in root/mods/noita-mp.

Reply:
Wait a sec. My bad. Looks fine so far.